frontend/src/container/WorkspaceContent.jsx

```diff
@@ -113,6 +125,18 @@ export class WorkspaceContent extends React.Component {
     this.updateUrlTitleBreadcrumbs()
   }
 
+  handleContentModified = data => {
+    if (data.author.user_id === this.props.user.userId) {
+      this.props.dispatch(addWorkspaceReadStatus(data.content, data.workspace.workspace_id))
```

Comment:
couldn't this be in the ReduxTlmDispatcher ?
Is there a reason why you put it in WorkspaceContent component ?

Reply:
Yes it should be in `ReduxTlmDispatcher`